src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/BucketController.java

```diff
@@ -64,8 +67,10 @@
     @RequestMapping(value = "/buckets", method = POST)
     @ResponseStatus(HttpStatus.CREATED)
     public BucketMetadata create(
-            @RequestParam(value = "name", required = true) String bucketName) {
-        return bucketService.createBucket(bucketName);
+            @RequestParam(value = "name", required = true) String bucketName,
+            @ApiParam(value = "The name of the user that owns the Bucket")
+            @RequestParam(value = "owner") Optional<String> ownerName) {
```

Comment:
You're right, and I'm going to run into trouble once I start to work on unicity of bucketName/owner